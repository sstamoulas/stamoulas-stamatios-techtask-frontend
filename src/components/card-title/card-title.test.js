import React from 'react'
import { create } from 'react-test-renderer'
 
import CardTitle from './card-title.component'

describe('Card Title Component', () => {
  let component
 
  beforeEach(() => { 
    component = create(<CardTitle title='Potato Salad' />)
  })
 
  it('should render with given props', () => {
    expect(component.toJSON()).toMatchSnapshot()
  })
})
