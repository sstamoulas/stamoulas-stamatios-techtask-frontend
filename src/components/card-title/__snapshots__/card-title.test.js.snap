// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Title Component should render with given props 1`] = `
<div
  className="flip-card-front"
>
  <div>
    Potato Salad
  </div>
</div>
`;
